docs/tutorial/index.rst

```diff
@@ -1,4 +1,147 @@
+parallel but you're also going to want to find time for some social events.
+
+You have organised your time slots as follows:
+
+- The first day will have 2 sessions (morning and afternoon) with 2 30 minute
+  time slots in each room.
+- The second day will have 1 room used for longer 1 hour workshops, the other
+  room used for more talks and 2 long sessions set aside for the social events.
+
+Let us create these time slots using the :code:`conference_scheduler`::
+
+    >>> from conference_scheduler.resources import Room, Slot, Event
+
+    >>> rooms = [Room(name="Big", capacity="200", suitability=[]),
+    ...          Room(name="Small", capacity="50", suitability=[]),
+    ...          Room(name="outside", capacity="1000", suitability=[])]
+    >>> talk_slots  = [Slot(room=rooms[0], starts_at='15-Sep-2016 09:30', duration=30, session="A", capacity=200),
+    ...                Slot(room=rooms[0], starts_at='15-Sep-2016 10:00', duration=30, session="A", capacity=200),
+    ...                Slot(room=rooms[1], starts_at='15-Sep-2016 09:30', duration=30, session="B", capacity=50),
+    ...                Slot(room=rooms[1], starts_at='15-Sep-2016 10:00', duration=30, session="B", capacity=50),
+    ...                Slot(room=rooms[0], starts_at='15-Sep-2016 12:30', duration=30, session="C", capacity=200),
+    ...                Slot(room=rooms[0], starts_at='15-Sep-2016 13:00', duration=30, session="C", capacity=200),
+    ...                Slot(room=rooms[1], starts_at='15-Sep-2016 12:30', duration=30, session="D", capacity=50),
+    ...                Slot(room=rooms[1], starts_at='15-Sep-2016 13:00', duration=30, session="D", capacity=50),
+    ...                Slot(room=rooms[0], starts_at='16-Sep-2016 09:30', duration=30, session="E", capacity=50),
+    ...                Slot(room=rooms[0], starts_at='16-Sep-2016 10:00', duration=30, session="E", capacity=50),
+    ...                Slot(room=rooms[0], starts_at='16-Sep-2016 12:30', duration=30, session="F", capacity=50),
+    ...                Slot(room=rooms[0], starts_at='16-Sep-2016 13:00', duration=30, session="F", capacity=50)]
+    >>> workshop_slots = [Slot(room=rooms[1], starts_at='16-Sep-2016 09:30', duration=60, session="G", capacity=50),
+    ...                   Slot(room=rooms[1], starts_at='16-Sep-2016 13:00', duration=60, session="H", capacity=50)]
+    >>> outside_slots = [Slot(room=rooms[2], starts_at='16-Sep-2016 12:30', duration=90, session="I", capacity=1000),
+    ...                  Slot(room=rooms[2], starts_at='16-Sep-2016 13:00', duration=90, session="J", capacity=1000)]
+    >>> slots = talk_slots + workshop_slots + outside_slots
+
+
+We also have a number of talks and workshops to schedule, because of the
+duration/location of the slots we know some of them are unavailable for a given slot::
+
+    >>> events = [Event(name='Talk 1', event_type='talk', duration=30, roles="empty", tags=['beginner'], unavailability=outside_slots, demand=50),
```

Comment:
It's intended (later) to capture the fact that a person might be a speaker, a mentor, a chair or some other role for an event.